src/engine/sys/sdl_glimp.cpp

```diff
@@ -1703,6 +1703,17 @@ static rserr_t GLimp_StartDriverAndSetMode( int mode, bool fullscreen, bool bord
 {
 	int numDisplays;
 
+#if !defined(_WIN32) && !defined(__APPLE__)
+	/* SDL2: Let X11 and Wayland desktops (Linux, FreeBSD…) display the game
+	name on the game windows instead of the engine binary name. This also
+	makes possible to display the correct icon if that WMCLASS is also set
+	in the net.unvanquished.Unvanquished.desktop file. */
+	Sys::SetEnv( "SDL_VIDEO_X11_WMCLASS", PRODUCT_NAME );
```

Comment:
These should also be PRODUCT_APPID

Reply:
Oh lol, I did not noticed that the default WMCLASS was the APPID, which makes sense btw.

So now all those hints are using `PRODUCT_APPID`.

This has the side effect of displaying `net.unvanquished.Unvanquished` in the taskbar if there is no XDG icon though.